server/routerlicious/packages/services-client/api-report/server-services-client.api.md

```diff
@@ -379,6 +379,7 @@ export interface INetworkErrorDetails {
     message?: string;
     retryAfter?: number;
     retryAfterMs?: number;
+    source?: string;
```

Comment:
'source' is a generic string, which could either be a message or a stringified stack trace.
In oss repo, we are creating networkError for http requests exceptions, and I already set this field to include information about the http url that returned the error, because there is no stack trace in http requests.
Will update it from frs also once this is merged. Let me know if I missed something.